snakefile

```diff
@@ -46,13 +46,25 @@ rule extract_adm0_gadm:
         "curl {params} -o {output} -O -J -L"
 
 rule extract_world_gadm:
+    # Note. By removing the output Snakemake will always re-run the process
+    # this means we could program a test into the extract process if
+    # necessary.
```

Comment:
You can force a snake rule to run even if the output exists by including the `-R` parameter. It's better to leave the output in because a) the rule will check for the existance of the output and complain if it's not found, b) in prinicple it should be passed to the Python entry point as a variable that is used to name the output, as is done in e.g. the COD admin extraction methods

Reply:
OK, thanks for clarifying the principles.